python_modules/libraries/dagster-celery/dagster_celery_tests/repo.py

```diff
@@ -8,14 +8,20 @@
     OutputDefinition,
     RetryRequested,
     default_executors,
+    fs_io_manager,
     lambda_solid,
     pipeline,
     solid,
 )
 from dagster.core.test_utils import nesting_composite_pipeline
 from dagster_celery import celery_executor
 
-celery_mode_defs = [ModeDefinition(executor_defs=default_executors + [celery_executor])]
+celery_mode_defs = [
+    ModeDefinition(
+        executor_defs=default_executors + [celery_executor],
+        resource_defs={"io_manager": fs_io_manager},
```

Comment:
The default IO manager now writes to the filesystem (if not called from execute_in_process), so this might not be needed?

Reply:
thought that was just for jobs (?) - for pipelines do users still need to provide fs_io_manager in this case?

Reply:
Ah, you are right